src/VPolygon.jl

```diff
@@ -553,3 +553,52 @@ function translate(P::VPolygon{N}, v::AbstractVector{N}) where {N<:Real}
                                 "set by a $(length(v))-dimensional vector"
     return VPolygon([x + v for x in vertices_list(P)])
 end
+"""
```

Comment:
Add an empty line between lines 556 and 557 to keep the different functions separated at least by one new line.